### DISCUSSION TITLE
Improve the visibility and design of the "Star us on GitHub" button.

### DISCUSSION THREAD

**Discussion by SCR01** [rustdesk/rustdesk#9533] [General]
**Proposed Changes:**

- Enhance Button Size: Increase the size of the button slightly to make it more prominent without overshadowing other elements on the page.

- Color Contrast: Use a more eye-catching color for the button that contrasts better with the surrounding elements (e.g., a brighter or complementary color).

- Add Animation: Consider adding subtle animation (such as a hover effect) to draw attention to the button when users scroll over it.

- Positioning: Consider placing the button closer to the core CTA (e.g., near the "Download" button) or in a floating header bar, making it more accessible as users scroll.

Impact: Improving this button's design could lead to higher engagement with GitHub stars, encouraging more developers to contribute to the project.

![Screenshot 2024-10-01 062753](https://github.com/user-attachments/assets/0701857e-77c6-426b-b00f-f47c661066a0)

**Comment by rustdesk** [comment]
Thanks, do you have an example?
